src/main/java/frc/robot/subsystems/Climber.java

```diff
@@ -279,16 +289,13 @@ public void setCurrentLimit(Current current) {
   }
 
   public Command climb() {
-    return Commands.sequence(
-        runClimber(ClimberConstants.kClimberAtCageSetpoint, 0),
-        Commands.waitUntil(isClimberAtPosition(ClimberConstants.kClimberAtCageSetpoint)),
-        runClimber(ClimberConstants.kClimberExtendedSetpoint, 1));
+    return Commands.sequence(extendToCage(), engageServo(), extendFully());
   }
 
   public Command retract() {
-    return runClimber(ClimberConstants.kClimberEmergencyUndoAngle, 0)
+    return disengageServo()
+        .andThen(runClimber(ClimberConstants.kClimberEmergencyUndoAngle, 0))
```

Comment:
The climber needs to move to a position where the servos can disengage, so it needs to move first.